com.wings3d.WINGS.json

```diff
@@ -144,9 +154,8 @@
       "sources": [
         {
           "type": "git",
-          "url": "https://github.com/eigenteam/eigen-git-mirror.git",
-          "tag": "3.3.7",
-          "commit": "cf794d3b741a6278df169e58461f8529f43bce5d"
+          "url": "https://gitlab.com/libeigen/eigen.git",
+          "tag": "3.3.9"
         }
```

Comment:
Have you tested that these changes work? 
Because we build linux (and windows and MacOS) with the "tag": "3.3.7",